src/librustc_resolve/lib.rs

```diff
@@ -950,6 +950,10 @@ enum RibKind<'a> {
     /// upvars).
     TraitOrImplItemRibKind,
 
+    /// We passed through a function definition. Disallow upvars.
+    /// Permit only those const parameters specified in the function's generics.
```

Comment:
Nit: The grammar looks a bit weird.
"only const parameters" or "only those const parameters that are specified"?

Reply:
The latter one; I've updated the comment.